amit/index.js

```diff
@@ -0,0 +1,30 @@
+gitlet dogIsBetter = true;
+let catIsBetter = false;
+// 1. a,b,c
+console.log(dogIsBetter && catIsBetter);
+
+console.log(dogIsBetter || catIsBetter);
+
+console.log(!(dogIsBetter && catIsBetter));
+
+// 2
+let atoms = 30;
+let sandGrains = 20;
+let starsInSky = 10;
+// ans 3 a
+console.log("ans. a ==>", atoms > (sandGrains, starsInSky));
```

Comment:
Better to use && symbol and solve the exercise like this:
console.log(`Q3a: ${(atoms > starsInSky) && (atoms > sandGrains)}`);

Reply:
console.log(Q3a:  ${(atoms > starsInSky)  &&  (atoms > sandGrains)} );
if i write this line i am getting error,
so i write lick this console.log("Q3a:", atoms > starsInSky && atoms > sandGrains );